.gitignore

```diff
@@ -1,2 +1,5 @@
 /node_modules
 /.proof.out
+
+# stryker temp files
+.stryker-tmp
```

Comment:
In case you don't plan to commit the generated report you need to also ignore `/reports`. (At least this is a folder that was generated when I ran it earlier.)

Reply:
I just pushed some of the changes you had originally made for PR #51, which would probably overlap #73 now.